### DISCUSSION TITLE
v24.07.24

### DISCUSSION THREAD

**Discussion by davidrapan** [davidrapan/ha-solarman#34] [Announcements]
## Scheduled release
* Added inverter configuration and improved fetching flow

## What's Changed
* Update api.py by @CrazyUs3r in https://github.com/davidrapan/ha-solarman/pull/8
* Update services.py by @CrazyUs3r in https://github.com/davidrapan/ha-solarman/pull/7
* Update services.yaml by @CrazyUs3r in https://github.com/davidrapan/ha-solarman/pull/6

## New Contributors
* @CrazyUs3r made their first contribution in https://github.com/davidrapan/ha-solarman/pull/8

**Full Changelog**: https://github.com/davidrapan/ha-solarman/compare/v24.07.14...v24.07.24

<hr /><em>This discussion was created from the release <a href='https://github.com/davidrapan/ha-solarman/releases/tag/v24.07.24'>v24.07.24</a>.</em>

**Comment by gedger** [comment]
Warning to anyone who uses a custom_parameters.yaml file for their inverter definition the update will delete it.

**Reply by PV-Joe** [reply]
Good to know, but that should not happen in my opinion.

@davidrapan
Could you have a look why this is happening?

**Comment by davidrapan** [comment]
Try it guys: [v24.07.25 pre-release](https://github.com/davidrapan/ha-solarman/releases/tag/v24.07.25)

It's required to move your custom profiles under that subdirectory though but I believe this separation is healthy change in general. ;)

**Reply by davidrapan** [reply]
It still somehow does not work.... :/ Dunno where the issue could be, guess I'll have to ask HAs community.

**Reply by davidrapan** [reply]
Okay I think I've got it now guys! (After many, many, many tries and errors :-D)
